tests/integration/tui/screenshots/test_auto_expansion.later-time.html

```diff
@@ -3,7 +3,7 @@
 <span style="color:#000000;background:#e5e5e5">-</span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5">~cylc                                                                         </span>
 <span style="color:#000000;background:#e5e5e5">   </span><span style="color:#000000;background:#e5e5e5"></span><span style="color:#e5e5e5;background:#000000">-</span><span style="color:#000000;background:#e5e5e5"></span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5;font-weight:bold">one</span><span style="color:#000000;background:#e5e5e5"> - </span><span style="color:#cdcd00;background:#e5e5e5">paused</span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#00cd00;background:#e5e5e5">1■</span><span style="color:#000000;background:#e5e5e5">                                                            </span>
 <span style="color:#000000;background:#e5e5e5">      </span><span style="color:#000000;background:#e5e5e5">-</span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5">● 1                                                                     </span>
-<span style="color:#000000;background:#e5e5e5">         </span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5">● b                                                                  </span>
+<span style="color:#000000;background:#e5e5e5">         </span><span style="color:#000000;background:#e5e5e5">+</span><span style="color:#000000;background:#e5e5e5"> </span><span style="color:#000000;background:#e5e5e5">● </span><span style="color:#00cd00;background:#e5e5e5">■</span><span style="color:#000000;background:#e5e5e5"> b                                                                </span>
```

Comment:
This test has nothing to do with "submit-failed" jobs so these tests breaking reveals a bug in the PR.

In this test, we run `cylc set` to mark a task as succeeded. Before this change, the test correctly showed the succeeded task with no job. After this change there is a succeeded job in the data which never logically existed.